dynamic-loader/main.c

```diff
@@ -0,0 +1,33 @@
+#include <msp430.h>
+#include <stdio.h>
+#include <string.h>
+#include <sancus_support/sm_io.h>
+#include <sancus_support/sm_control.h>
+#include <sancus_support/global_symtab.h>
+
+#include "sm1.h"
+
+int main()
+{
+    msp430_io_init();
+    printf("Loading SM from buffer..\n");
+    int id = sm_load((void *) SM_BINARY, SM_NAME, VENDOR_ID);
+    printf("Loaded SM id: %d\n", id);
```

Comment:
nice!

we can consider extending this later to also call the SM to verify it's indeed correctly loaded :thinking:

Reply:
done as per 4f48273